docs/source/pipelines.rst

```diff
@@ -0,0 +1,541 @@
+.. _pipelines:
+
+===============================
+MLflow Pipelines (experimental)
+===============================
+
+MLflow Pipelines is an opinionated framework for structuring MLOps workflows that simplifies and
+standardizes machine learning application development and productionization. MLflow Pipelines
+makes it easy for data scientists to follow best practices for creating production-ready ML
+deliverables, allowing them to focus on utilizing their expert knowledge to develop excellent
+models. MLflow Pipelines also enables ML engineers and DevOps teams to seamlessly deploy these
+models to production and incorporate them into applications.
+
+MLflow Pipelines provides predefined production-quality pipeline templates for common ML problem
+types, such as regression & classification, and MLOps tasks, such as batch scoring. Pipelines are
+structured as git repositories with YAML-based configuration files and Python code, offering
+developers a declarative approach to ML application development that reduces boilerplate.
+
+MLflow Pipelines also implements a cache-aware executor for :ref:`Pipeline Steps
+<steps-key-concept>`, ensuring that steps are only executed when associated :py:ref:`code or
+configurations <pipeline-repositories-key-concept>` have changed. This enables data scientists,
+ML engineers, and DevOps teams to iterate very quickly within their domains of expertise. MLflow
+offers :ref:`APIs <pipelines-apis>` and a :ref:`CLI <cli>` for executing pipelines and inspecting
+their results.
+
+Installation
+------------
+MLflow Pipelines is available as an extension of the
+`MLflow Python library <https://pypi.org/project/mlflow/>`_. You can install MLflow Pipelines
+from PyPI as follows:
+
+.. code-block:: sh
+
+  pip install mlflow[pipelines]
+
+.. note::
+  You can install MLflow Pipelines from a Databricks Notebook by running
+  ``%pip install mlflow[pipelines]`` or install MLflow Pipelines on a Databricks Cluster
+  by following the instructions at
+  https://docs.databricks.com/libraries/cluster-libraries.html#install-a-library-on-a-cluster.
+
+  Note that `Databricks Runtime <https://docs.databricks.com/runtime/dbr.html>`_ version 11.0
+  or greater is required in order to install MLflow Pipelines on Databricks.
+
+
+Key Concepts
+------------
+
+.. _steps-key-concept:
+
+Steps
+~~~~~
+
+Steps represent individual modeling or MLOps procedures, such as ingesting data,
```

Comment:
```suggestion
Steps represent individual MLOps procedures, such as ingesting data,
```

Reply:
MLOps is distinct from model development, so I'd prefer to keep `modeling` in there:

```
MLOps or ML Ops is a set of practices that aims to deploy and maintain machine learning models in production reliably and efficiently. The word is a compound of "machine learning" and the continuous development practice of DevOps in the software field.
```

https://en.wikipedia.org/wiki/MLOps

Reply:
TIL again lol